tools/RAiDER/utilFcns.py

```diff
@@ -844,33 +844,33 @@ def write2NETCDF4core(nc_outfile, dimension_dict, dataset_dict, tran, mapping_na
     from osgeo import osr
 
     if mapping_name == 'WGS84':
-
         epsg = 4326
         crs = pyproj.CRS.from_epsg(epsg)
 
         grid_mapping = 'WGS84'  # need to set this as an attribute for the image variables
-        datatype = np.dtype('S1')
-        dimensions = ()
+    else:
+        crs = pyproj.CRS.from_wkt(mapping_name)
+        grid_mapping = 'CRS'
 
-        var = nc_outfile.createVariable(
-            mapping_name,
-            datatype,
-            dimensions,
-            fill_value=None
-        )
-        # variable made, now add attributes
+    datatype = np.dtype('S1')
+    dimensions = ()
 
-        for k, v in crs.to_cf().items():
-            var.setncattr(k, v)
+    var = nc_outfile.createVariable(
+        grid_mapping,
+        datatype,
+        dimensions,
+        fill_value=None
+    )
 
-        # These might not be needed as to_cf adds a crs_wkt
-        var.setncattr('spatial_ref', crs.to_wkt())
-        var.setncattr('spatial_proj4', crs.to_proj4())
-        var.setncattr('spatial_epsg', epsg)
-        var.setncattr('GeoTransform', ' '.join(str(x) for x in tran))  # note this has pixel size in it - set  explicitly above
+    # variable made, now add attributes
+    for k, v in crs.to_cf().items():
+        var.setncattr(k, v)
 
-    else:
-        raise Exception('Grid mapping name not supported; currently, only WGS84 (EPSG: 4326) is supported!')
+    # These might not be needed as to_cf adds a crs_wkt
```

Comment:
Does this need maintained or is it verified that we don't need the commented lines?

Reply:
These can go